pkg/awx/connection.go

```diff
@@ -263,7 +305,7 @@ func (c *Connection) rawGet(path string, query url.Values) (output []byte, err e
 	if err != nil {
 		return
 	}
-	if response.StatusCode != 200 {
+	if response.StatusCode > 202 {
```

Comment:
don't we still want to check for other possible errors `!=200` ?
are codes 201-202 OK?
We might want to add a comment about that

Reply:
201 Created (we create tokens in this PR), 202 Accepted is what you sometimes get when you launch a background job. These (and 200 of course)are the most common in my experience.